<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>demandeenseignant</class>
 <widget class="QFrame" name="demandeenseignant">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 12pt &quot;Open Sans&quot;;</string>
  </property>
  <widget class="QGroupBox" name="EnvoyerDem">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>331</width>
     <height>751</height>
    </rect>
   </property>
   <property name="title">
    <string>Envoyer demandes</string>
   </property>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>311</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="NomMat">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>580</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom Materiel :</string>
    </property>
   </widget>
   <widget class="QLabel" name="nommatlabel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>580</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Quantite">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>610</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Quantité disponible :</string>
    </property>
   </widget>
   <widget class="QLabel" name="quantitelabel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>610</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="QuantiteDesiree">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>640</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Quantité désirée :</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="qtebox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>640</y>
      <width>42</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>710</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Commander</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>670</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Date d'envoi :</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>670</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="displayFormat">
     <string>dd/MM/yyyy</string>
    </property>
    <property name="calendarPopup">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>710</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Commander Arduino</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="DemandesGroup">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>581</width>
     <height>361</height>
    </rect>
   </property>
   <property name="title">
    <string>Voir requettes</string>
   </property>
   <widget class="QTableView" name="tableView_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>561</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>322</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Annuler</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Imprimer">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>380</y>
     <width>581</width>
     <height>381</height>
    </rect>
   </property>
   <property name="title">
    <string>Recus</string>
   </property>
   <widget class="QPushButton" name="imprimerbutton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>342</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Imprimer</string>
    </property>
   </widget>
   <widget class="QTableView" name="tablerecus">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>561</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>352</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>En attente...</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
